help/destinations/catalog/analytics/gainsight-px.md

```diff
@@ -0,0 +1,107 @@
+
+
+## Overview {#overview}
+
+[Gainsight PX](https://www.gainsight.com/product-experience/) is a product experience platform that enables product teams to understand how users use their products, collect feedback, and create in-app engagements like product walkthroughs to drive user onboarding and product adoption.
+
+>[!IMPORTANT]
+>
+>The destination connector and documentation page are created and maintained by the *Gainsight PX* team. For any inquiries or update requests, please contact them directly at *`pxsupport@gainsight.com`.*
+
+## Prerequisites {#prerequisites}
+
+* Contact Gainsight support to request the external segment features are enabled for your subscription
+* Generate an OAuth Secret value for your PX subscription, using the Generate New Secret button at the bottom of the Company Details page.
+
+## Supported identities {#supported-identities}
+
+
+*Gainsight PX* supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+| Target Identity |Description| Considerations                                                                                                                                                                                                                                                                         |
+|-----------------|---|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| IdentifyID      |Custom user IDs| Use this namespace to use whatever identify is used when identifying users within PX.                                                                                                                                                                                                  |
+
+{style="table-layout:auto"}
+
+## Supported audiences {#supported-audiences}
+
+This section describes which type of audiences you can export to this destination.
+
+| Audience origin | Supported | Description | 
+---------|----------|----------|
+| [!DNL Segmentation Service] | ✓ | Audiences generated through the Experience Platform [Segmentation Service](../../../segmentation/home.md).|
+| Custom uploads | X | Audiences [imported](../../../segmentation/ui/overview.md#import-audience) into Experience Platform from CSV files. |
+
+{style="table-layout:auto"}
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Segment export]** | You are exporting all members of a segment (audience) with the identifiers (name, phone number, or others) used in the *YourDestination* destination.|
```

Comment:
Please update the description of Export type to:  You are exporting all members of an audience with the identifiers (name, phone number, or others) used in the [!DNL Gainsight PX] destination.